etl/app/server/profiles/assessments.js

```diff
@@ -70,6 +71,218 @@ export const createQuery = `CREATE TABLE IF NOT EXISTS ${tableName}
     associatedactionagency VARCHAR(10)
   )`;
 
+export const createIndexes = `
+  CREATE INDEX IF NOT EXISTS ${indexTableName}_alternatelistingidentifier_asc
+    ON ${tableName} USING btree
+    (alternatelistingidentifier COLLATE pg_catalog."default" ASC NULLS LAST)
+    TABLESPACE pg_default;
+
+  CREATE INDEX IF NOT EXISTS ${indexTableName}_assessmentbasis_asc
+    ON ${tableName} USING btree
+    (assessmentbasis COLLATE pg_catalog."default" ASC NULLS LAST)
+    TABLESPACE pg_default;
+
+  CREATE INDEX IF NOT EXISTS ${indexTableName}_assessmentdate_desc
+    ON ${tableName} USING btree
+    (assessmentdate DESC NULLS LAST)
+    TABLESPACE pg_default;
+
+  CREATE INDEX IF NOT EXISTS ${indexTableName}_assessmentmethods_asc
+    ON ${tableName} USING btree
+    (assessmentmethods COLLATE pg_catalog."default" ASC NULLS LAST)
+    TABLESPACE pg_default;
+
+  CREATE INDEX IF NOT EXISTS ${indexTableName}_assessmenttypes_asc
+    ON ${tableName} USING btree
+    (assessmenttypes COLLATE pg_catalog."default" ASC NULLS LAST)
+    TABLESPACE pg_default;
+
+  CREATE INDEX IF NOT EXISTS ${indexTableName}_assessmentunitid_asc
+    ON ${tableName} USING btree
+    (assessmentunitid COLLATE pg_catalog."default" ASC NULLS LAST)
+    TABLESPACE pg_default;
+
+  CREATE INDEX IF NOT EXISTS ${indexTableName}_assessmentunitname_asc
+    ON ${tableName} USING btree
+    (assessmentunitname COLLATE pg_catalog."default" ASC NULLS LAST)
+    TABLESPACE pg_default;
+
+  CREATE INDEX IF NOT EXISTS ${indexTableName}_assessmentunitstatus_asc
+    ON ${tableName} USING btree
+    (assessmentunitstatus COLLATE pg_catalog."default" ASC NULLS LAST)
+    TABLESPACE pg_default;
+
+  CREATE INDEX IF NOT EXISTS ${indexTableName}_associatedactionagency_asc
+    ON ${tableName} USING btree
+    (associatedactionagency COLLATE pg_catalog."default" ASC NULLS LAST)
+    TABLESPACE pg_default;
+
+  CREATE INDEX IF NOT EXISTS ${indexTableName}_associatedactionid_asc
+    ON ${tableName} USING btree
+    (associatedactionid COLLATE pg_catalog."default" ASC NULLS LAST)
+    TABLESPACE pg_default;
+
+  CREATE INDEX IF NOT EXISTS ${indexTableName}_associatedactionname_asc
+    ON ${tableName} USING btree
+    (associatedactionname COLLATE pg_catalog."default" ASC NULLS LAST)
+    TABLESPACE pg_default;
+
+  CREATE INDEX IF NOT EXISTS ${indexTableName}_associatedactionstatus_asc
+    ON ${tableName} USING btree
+    (associatedactionstatus COLLATE pg_catalog."default" ASC NULLS LAST)
+    TABLESPACE pg_default;
+
+  CREATE INDEX IF NOT EXISTS ${indexTableName}_associatedactiontype_asc
+    ON ${tableName} USING btree
+    (associatedactiontype ASC NULLS LAST)
+    TABLESPACE pg_default;
+
```

Comment:
The `cyclefirstlisted` index is missing, not sure if ascending or descending is better in this case:
```suggestion
CREATE INDEX IF NOT EXISTS ${indexTableName}_cyclefirstlisted_desc
    ON ${tableName} USING btree
    (cyclefirstlisted COLLATE pg_catalog."default" DESC NULLS LAST)
    TABLESPACE pg_default;
```